rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :book, :person
```

Comment:
- [ ] Let's mark `book` and `person` with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ to it. Kindly ensure to improve upon this aspect.